src/main/zapHomeFiles/hudtutorial/en_GB/AttackMode.html

```diff
@@ -7,12 +7,12 @@
 <body>
 <div class="roundContainer">
 	<h1>Attack Mode</H1>
-	The 'Attack Mode' <img src="crosshairs-grey.png"> tool performs the same attacks as an Active Scan but works in a subtly different way.
+	The 'Attack Mode' <img src="crosshairs-grey.png" alt="[crosshairs]"> tool performs the same attacks as an Active Scan but works in a subtly different way.
 	<p>
 	Instead of attacking the URLs known at the time it is started, it attacks any URLs that are in scope as soon as they
-	are discovered. It does not show any progress because it stays active with the <img src="crosshairs.png"> icon until is is stopped.
+	are discovered. It does not show any progress because it stays active with the <img src="crosshairs.png"> icon until it is stopped.
```

Comment:
fixes #645

Reply:
Could you add to the PR description (to automatically close it)?